feat : add basic desc and configurations in readme (#13)
mojaloop/accounts-and-balances-bc: README.md

# Accounts and Balances Bounded Context

The Accounts and Balances BC acts as the “central ledger” for the system. It interacts primarily with the Settlements BC, Participants Lifecycle BC and Transfers BCs, and is a directed sub-system, which means that it is a dependency of the BCs that use it as a “financial system of record” for the financial accounting.

> See the Reference Architecture documentation [Thirdparty API](https://mojaloop.github.io/reference-architecture-doc/boundedContexts/thirdPartyApi/) for context on this vNext implementation guidelines.  

Reviewer: ```suggestion
See the Reference Architecture documentation of [Accounts and Balances BC](https://mojaloop.github.io/reference-architecture-doc/boundedContexts/accountsAndBalances) for context on this vNext implementation guidelines.  
```

 As part of our CI/CD process, we use CircleCI. The CircleCI workflow automates the process of publishing changed packages to the npm registry and building Docker images for select packages before publishing them to DockerHub. It also handles versioning, tagging commits, and pushing changes back to the repository.

The process includes five phases. 
1. Setup : This phase initializes the environment, loads common functions, and retrieves commits and git change history since the last successful CI build.

2. Detecting Changed Package.

3. Publishing Changed Packages to NPM.

4. Building Docker Images and Publishing to DockerHub.

5. Pushing Commits to Git.

 All code is automatically linted, built, and unit tested by CircleCI pipelines, where unit test results are kept for all runs. All libraries are automatically published to npm.js, and all Docker images are published to Docker Hub.

 ## Documentation
The following documentation provides insight into the FSP Interoperability API Bounded Context.

- **Reference Architecture** - https://mojaloop.github.io/reference-architecture-doc/boundedContexts/accountsAndBalances/
- **MIRO Board** - https://miro.com/app/board/o9J_lJyA1TA=/

> - **Work Sessions** - https://docs.google.com/document/d/1Nm6B_tSR1mOM0LEzxZ9uQnGwXkruBeYB2slgYK1Kflo/edit#heading=h.6w64vxvw6er4

Reviewer: If the intention is to keep using these links and resources - Miro and Google docs, we need to move these to GitHub. Using these risk losing access to any relevant documentation or losing them altogether if the plans expire or accounts are deleted..
